### PR TITLE
Support Python 3.13

### PR DESCRIPTION
**Pull request by edgarrmondragon**
## What do these changes do?

Add support and build wheels for Python 3.13

Context for test updates:

- https://docs.python.org/3.13/whatsnew/3.13.html#new-features
- [`class.__static_attributes__`](https://docs.python.org/3.13/library/stdtypes.html#class.__static_attributes__)

## Are there changes in behavior for the user?

No

## Related issue number

- https://github.com/aio-libs/frozenlist/issues/588

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] If you provide code modifications, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep the list in alphabetical order, the file is sorted by name.
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

**Comment by edgarrmondragon** [comment]
Docs failure is https://github.com/sphinx-contrib/sphinxcontrib-towncrier/issues/92

File: requirements/ci.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 -r towncrier.txt
 build==1.0.3
-coverage==7.3.3
+coverage==7.6.1
```
Comment (bdraco) [inline requirements/ci.txt:3]: Would you please bump this in a separate PR

Comment (edgarrmondragon) [reply requirements/ci.txt:3]: Done: https://github.com/aio-libs/frozenlist/pull/597

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks @edgarrmondragon

**Comment by edgarrmondragon** [comment]
@bdraco sorry for the ping. Are there any blockers for a release of this to PyPI?

**Pull request merged**